packages/components/src/slot-fill/bubbles-virtually/fill.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * WordPress dependencies
+ */
+import { useRef, useEffect, createPortal } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import useSlot from './use-slot';
+
+export default function Fill( { name, children } ) {
+	const slot = useSlot( name );
+	const ref = useRef();
+
+	useEffect( () => {
+		slot.registerFill( ref );
+		return () => {
+			slot.unregisterFill( ref );
+		};
+	}, [ slot.registerFill, slot.unregisterFill ] );
```

Comment:
This is just so we can keep track of fills and check their existence later (on `BlockInspector`, for example).